external-crates/move/move-compiler/tests/move_check_testsuite.rs

```diff
@@ -79,6 +81,28 @@ fn move_check_testsuite(path: &Path) -> datatest_stable::Result<()> {
             Path::new(&verification_exp_path),
             Path::new(&verification_out_path),
             Flags::verification(),
+            true,
+        )?;
+    }
+
+    // A cross-module unused case that should run without unused warnings suppression
+    if path.with_extension(UNUSED_EXT).exists() {
```

Comment:
This feels not so great... especially that it just feels a bit overly specified for the issue we are running into now. 

How much work would it be to change the unused functions errors in the tests? 
Like, how many tests is it?

Reply:
It's over 60% of the tests... Otherwise I wouldn't bother.